### ISSUE TITLE
Replace MELPA version with this one?

### ISSUE THREAD

**Issue by ddoherty03** [vedang/pdf-tools#10]
I notice the MELPA version of pdf-tools is still pointing to the repo "politza/pdf-tools".

Have you guys worked out transferring maintenance from there to here?  I keep getting bit by the

'Symbol’s function definition is void: pdf-misc-print-program' error. 

Your README suggests that this can be installed via MELPA.  Could you at least clarify that?

Thanks for taking this important package up.

**Comment by vedang** [comment]
Hello @ddoherty03 

Thanks for creating this issue, I'll use it to track the MELPA related change. I have to raise this with MELPA maintainers and ask them to make the change, but have been putting it off due to <life stuff>

Thanks for the reminder!

**Comment by thisch** [comment]
FYI as a temporary workaround the quelpa tool can be used for installing vedang's pdf-tools version:

```el
(use-package pdf-tools
  :quelpa (pdf-tools :fetcher github :repo "vedang/pdf-tools")
  :commands (pdf-view-mode) ;; otherwise void-function pdf-view-mode is
                                                ;; raised when a PDF file is opened
  :ensure t
)
```

**Comment by ddoherty03** [comment]
Thanks @vedang for noting.  The @thisch suggestion might be a good thing to put in the README in the meantime.  Thanks, @thisch for the suggestion.  I have it on manual install for now, though.

**Comment by vedang** [comment]
This is now fixed with melpa/melpa#7567 , you should be able to install directly from MELPA. Closing this ticket.

**Comment by ddoherty03** [comment]
@vedang, many thanks.  Works perfectly from melpa.
